### ISSUE TITLE
define-unparser not provided externally

### ISSUE THREAD

**Issue by schuster** [nanopass/nanopass-framework-racket#3]
Judging from a quick look through the code, it looks like `define-parser` and `define-unparser` are the primary ways to parse and unparse languages. The main file provides `define-parser`, but not `define-unparser`. Any particular reason why not both?

**Comment by schuster** [comment]
Oh, hmm, apparently `define-language` still defines the unparser named after its defined language, but not the parser. That asymmetry is confusing.

**Comment by LeifAndersen** [comment]
I agree. @akeep Do you think it would make sense to make it consistent?

**Comment by akeep** [comment]
Yes, absolutely.  In fact, I thought it already was, though it appears the not in the Racket version.

-andy:)

On March 10, 2016 at 4:51:34 PM, Leif Andersen (notifications@github.com) wrote:

I agree. @akeep Do you think it would make sense to make it consistent?
